app/Http/Controllers/EntryController.php

```diff
@@ -15,7 +16,13 @@ class EntryController extends Controller
      */
     public function index(Request $request)
     {
-        $entries = Entry::all();
+        if ($userId = $request->get('user_id')) {
+            $entries = Entry::where('user_id', $userId)->get();
+        } elseif (is_null($request->get('user_id'))) {
+            $entries = Entry::where('user_id', Auth::user()->id)->get();
+        } else {
+            $entries = [];
+        }
```

Comment:
i don't think this condition is reachable since `user_id` is either present or not